### ISSUE TITLE
DOCS: Guides out of date in multiple languages

### ISSUE THREAD

**Issue by BootsinSoots** [ArchipelagoMW/Archipelago#2295]
### What task needs to be completed?

MrPokemon11 went through the guides today after noticing errors in non-english guides. He produced the following list (acceptable guides removed)

-Advanced settings
  explain what "BK mode" is?
  otherwise seems fine

-Triggers guide
  Seems fine, maybe include that it fully overwrites the option it changes?

-Plando guide
  add OoT to list of games that have connection plando

-Adventure
  Has two sections labeled "Multiworld Setup Guide" on the main tutorial page, except one is in french
  otherwise seems fine

-Clique
  no comment

-DS3
  I recommend changing the background color, as red text on a green background may be hard for colorblind people to read
  otherwise it seems fine

-DKC3
  I don't think including the instructions on how to host is necessary for this guide as it's covered elsewhere

-Factorio
  wordy, may be confusing to new people (certainly was to me when i was setting it up)

-FF1
  Add something that says to write down what the slot name is, considering how easily it can be lost

-HK
   change Scarab+ to Lumafly (even though the link redirects automatically, it could still be confusing)

-KH2
  add "save in file that isn't the first one it shows you upon starting a new game" to the best practices to prevent accidentally overriding things like async save files

-ALttP
  Completely rewrite the french and spanish translations, they're 2+ years out of date
  deutsch translation needs refurbishment (installation section especially imo)
  (MSU-1) discrepancy between english and spanish/french tutorial
  (Plando) use a different phrase than "throws a coin" (under the sewer image)
  (OOF replacement) i have no idea if this is still possible, someone else look at it

-Minecraft
  spanish and swedish translations include outdated yaml examples
  English and french seem to be okay

-OoT
  No comment of substance

-SoE
  the A in "page" under "where do i get a config file?" is capitalized when it shouldnt be
  probably needs some refurbishment

-SMZ3
  probably needs refurbishment

-SM
  has macintosh setup been figured out yet? if so add it
  Seems fine, i do recommend refurbishment

-Undertale
  i recommend improving the readability of the tutorial, otherwise it seems fine

-Witness
  out of date, game now has in-game messages (already has PR to fix)

**Comment by Silvris** [comment]
Oof replacement was introduced in 0.4.1, so I would be shocked if it managed to be out-of-date that quickly.

**Comment by BootsinSoots** [comment]
Let me know when you do so I can link it here. I suppose I probably shouldn't mark these as checked with the PRs not pulled

**Comment by ShinyNT** [comment]
regarding the Adventure point, I brought it up on the #archipelago-dev discord channel because it isn't necessarily something that needs to be corrected
tl;dr there isn't a clear standard on how that should be dealt with, either collapsing the Adventure section to follow the other guides format, or uncollapse eveything else so each guide can have a tittle in their own language

https://discord.com/channels/731205301247803413/731214280439103580/1161808319892160553 link to the discussion for those interested

**Comment by nicholassaylor** [comment]
Took a look into the DS3 documentation and saw that the red text was present in both languages. Looking at the doc using a Protanopia filter (the colorblindness most relevant here), the text looks as such.
![image](https://github.com/ArchipelagoMW/Archipelago/assets/79181893/49fd7464-67de-46b9-9ce1-e5237f908d19)

Playing around with values using the WCAG guidelines provided within the accessibility tab of most web browser dev tools, the best value I could get was essentially identical to that of the link colors. While this might cause confusion for some people, it may be the best choice for making this text accessible. However, I would like more input on this topic.

**Comment by BootsinSoots** [comment]
Wow, that is pretty invisible, huh? If it has to be a specific blue, then I guess that's the way to go. I'm not familiar with tools for checking against potential colorblindness issues

**Comment by Marechal-L** [comment]
In DS3, the french setup guide is missing downpatching instructions.
I'll provide a PR to translate it
